common/ayon_common/distribution/file_handler.py

```diff
@@ -112,27 +118,27 @@ def download_url(
         try:
             print(f"Downloading {url} to {fpath}")
             RemoteFileHandler._urlretrieve(url, fpath, headers=headers)
-        except (urllib.error.URLError, IOError) as e:
-            if url[:5] == "https":
-                url = url.replace("https:", "http:")
-                print((
-                    "Failed download. Trying https -> http instead."
-                    f" Downloading {url} to {fpath}"
-                ))
-                RemoteFileHandler._urlretrieve(url, fpath,
-                                               headers=headers)
-            else:
-                raise e
+        except (urllib.error.URLError, IOError) as exc:
+            if url[:5] != "https":
```

Comment:
Maybe we could make `_get_redirect_url` return a tuple of `(protocol, url)` -> `("https", "https://ayon.cloud/bla/bleh")` so we can avoid this kind of string slices

Reply:
This is actually for cases when redirect won't happen inside `_get_redirect_url`. So not really...

Reply:
I was referring to do this for line `108` so that then when we reach this point, we already have the protocol in a variable, and we don't need to do `url[:5]` since we already had `urllib` do it for us (it could be empty for all I know)

Reply:
I actually don't understand why this is here. It would crash in `_get_redirect_url` if the url would be wrong...